addon/components/o-s-s/currency-input.hbs

```diff
@@ -1,16 +1,28 @@
 <div class="currency-input-container fx-1" ...attributes>
-  <div class="currency-input upf-input fx-row fx-1 fx-xalign-center">
-    <div class="currency-selector fx-row" role="button" {{on "click" this.toggleCurrencySelector}}>
-      <span class="margin-right-xxx-sm text-center symbol">{{this.selectedCurrencySymbol}}</span>
+  <div class="{{if this.onlyCurrencyInput "currency-input-onlycurrency" "currency-input"}} upf-input fx-row fx-1 fx-xalign-center ">
+    <div class="currency-selector fx-row fx-malign-space-between" role="button" {{on "click" this.toggleCurrencySelector}}>
+      <div class="fx-col">
+        <div class="fx-row fx-gap-px-24">
+          <span class="text-center symbol">{{this.selectedCurrencySymbol}}</span>
+          {{#if this.onlyCurrencyInput}}
+            <span class=" margin-right-px-24 text-center symbol">{{this.selectedCurrencyCode}}</span>
+          {{/if}}
+        </div>
+      </div>
+      <div class="fx-col">
```

Comment:
This feels a bit off. Why are you adding these divs ?

Why are you using `text-center` & `symbol` and what's `symbol` ?